### PR TITLE
docs: refactor Popover examples markup and styles

### PR DESCRIPTION
**Pull request by rolfsmeds**

### CODE REVIEW

File: frontend/demo/component/popover/react/popover-notification-panel.tsx [hunk 1]
```diff
@@ -71,33 +71,31 @@ function Example() {
...
         </HorizontalLayout>
-        <TabSheet theme="no-padding">
+        <TabSheet theme="small no-padding" className="notifications">
           <TabSheetTab label="Unread">
             {unreadNotifications.value.length ? (
-              <MessageList
-                items={unreadNotifications.value}
-                className="notifications"
-              ></MessageList>
+              <MessageList items={unreadNotifications.value}></MessageList>
```
Comment (web-padawan) [inline frontend/demo/component/popover/react/popover-notification-panel.tsx:92]:
```suggestion
              <MessageList items={unreadNotifications.value} />
```

File: frontend/demo/component/popover/react/popover-notification-panel.tsx [hunk 2]
```diff
@@ -71,33 +71,31 @@ function Example() {
...
             ) : (
-              'No new notifications'
+              <div className="no-notifications-msg">No new notifications</div>
             )}
           </TabSheetTab>
           <TabSheetTab label="All">
-            <MessageList items={allNotifications.value} className="notifications"></MessageList>
+            <MessageList items={allNotifications.value}></MessageList>
```
Comment (web-padawan) [inline frontend/demo/component/popover/react/popover-notification-panel.tsx:98]:
```suggestion
            <MessageList items={allNotifications.value} />
```

File: frontend/demo/component/popover/react/popover-anchored-dialog.tsx [hunk 3]
```diff
@@ -68,15 +68,15 @@ function Example() {
             ></Checkbox>
           ))}
         </CheckboxGroup>
-        <HorizontalLayout theme="spacing-xs">
-          <Button
+        <HorizontalLayout style={{ justifyContent: 'space-between' }}>
+          <Button theme='small'
```
Comment (web-padawan) [inline frontend/demo/component/popover/react/popover-anchored-dialog.tsx:72]:
```suggestion
          <Button 
            theme='small'
```

File: frontend/demo/component/popover/react/popover-anchored-dialog.tsx [hunk 4]
```diff
@@ -68,15 +68,15 @@ function Example() {
...
             onClick={() => {
               columns.value = columns.value.map((column) => ({ ...column, visible: true }));
             }}
           >
             Show all
           </Button>
-          <Button
+          <Button theme='small'
```
Comment (web-padawan) [inline frontend/demo/component/popover/react/popover-anchored-dialog.tsx:79]:
```suggestion
          <Button 
            theme='small'
```

File: frontend/demo/component/popover/popover-user-menu.ts [hunk 5]
```diff
@@ -67,28 +68,32 @@ export class Example extends LitElement {
 
     return html`
       <vaadin-horizontal-layout
-        theme="spacing"
-        style="background: var(--lumo-contrast-5pct); padding: var(--lumo-space-s); margin: calc(var(--lumo-space-s) * -1)"
+        style="background: var(--lumo-contrast-5pct);
```
Comment (web-padawan) [inline frontend/demo/component/popover/popover-user-menu.ts:71]: IMO we should extract these to `.css` file, it's a bit too much for inline styles.

Comment (rolfsmeds) [reply frontend/demo/component/popover/popover-user-menu.ts]: Refactored now in latest commit.

**Review by web-padawan — APPROVED** [review]
LGTM. Pushed some Prettier code formatting tweaks and added missing `setPosition()` to the `PopoverUserMenu` example (it was only present in WC and React examples).

**Pull request merged**